seed.js

```diff
@@ -0,0 +1,90 @@
+const { users, categories, thread, comments } = require('./models')
+
+const main = async () => {
+    
+    
+
+    // Create users
+    await users.create({
+        id: 1,
```

Comment:
I dont think you need to give anything an id. I think when we run the database it will automatically give it a primary key

Reply:
Ok deleted for next push.